scripts/scylla-entry.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "skip_wait_for_gossip_to_settle: 0" >> /etc/scylla/scylla.yaml
```

Comment:
I vivedly remember testing this feature, time to time it would produce broken cluster, please remove it.

Reply:
It's already used inside SCT for integration tests.

If it can lead to issues, I can remove it, but GH Actions would be much slower

Reply:
Can you try to run without it, but on `6.1.1`, it is way faster

Reply:
I'll try